configure.ac

```diff
@@ -32,6 +32,8 @@ BOOST_REGEX
 BOOST_FOREACH
 BOOST_SYSTEM
 BOOST_FILESYSTEM
+AM_CONDITIONAL([HAVE_BOOST_GE_148], [test "$boost_major_version" -ge 148])
+PDNS_ENABLE_UNIT_TESTS
```

Comment:
rename to `WFORCE`?

Reply:
Maybe but I have a bunch of these from pdns, and I update them on a regular basis so it would be a pita